src/commands/prompt/index.ts

```diff
@@ -48,7 +49,13 @@ export const command: CommandModule<{}, PromptOptions> = {
         default: false,
         describe: 'Verbose output',
       }),
-  handler: (args) => run(args._.join(' '), args),
+  handler: (args) => {
```

Comment:
Instead of trying to handle it here, you should modify `bin.ts` to include init command as well.